src/Avalonia.NameGenerator.Sandbox/Avalonia.NameGenerator.Sandbox.csproj

```diff
@@ -6,10 +6,10 @@
         <IsPackable>false</IsPackable>
     </PropertyGroup>
     <ItemGroup>
-        <PackageReference Include="Avalonia" Version="0.10.0-preview5" />
-        <PackageReference Include="Avalonia.Desktop" Version="0.10.0-preview5" />
-        <PackageReference Include="Avalonia.ReactiveUI" Version="0.10.0-preview5" />
-        <PackageReference Include="Citrus.Avalonia" Version="1.2.6" />
+        <PackageReference Include="Avalonia" Version="0.10.999-cibuild0012694-beta" />
```

Comment:
Wouldn't it be better to set up and using the variable in `src/Directory.build.props` with the following content?

```xml
<PropertyGroup>
    <AvaloniaVersion>0.10.999-cibuild0012694-beta</AvaloniaVersion>
</PropertyGroup>
```

Reply:
You are totally right, thanks! Going to incorporate this pattern soon.

Reply:
Hmm. The issue here is, that the name reference generator itself doesn't require Avalonia to be preview. The original idea was to use different Avalonia versions in the generator and in sandbox in order to avoid bumping Avalonia version requirement for name reference generator users (e.g. if that generator uses a `preview` version, the generator won't be compatible with `0.10.0`). But yeah, probably worth using a constant instead of copy-pasting version strings.

Reply:
Ah, we aren't referencing Avalonia in the main source project.

Fine then, worth moving the version variable to `Directory.build.props`.